HedgeModManager/Config/RegistryConfig.cs

```diff
@@ -12,13 +12,15 @@ public static class RegistryConfig
         private const string PersonalizePath = @"SOFTWARE\Microsoft\Windows\CurrentVersion\Themes\Personalize";
 
         public static string ConfigPath { get; } = @"SOFTWARE\HEDGEMM";
-        public static string LastGameDirectory;
+        public static string LastGame;
```

Comment:
Not sure what the reason for this change is, renaming the field doesn't tell the reader what this property does.

Reply:
It was changed so it could be referred to with `nameof` without breaking the pre-existing registry value (which is named `LastGame`, not `LastGameDirectory`), but I can revert this change if necessary.

Alternatively, reflection could be used here instead to get public static fields in the `RegistryConfig` class.

It wouldn't be completely disastrous if the original value name was lost anyway, would just require the user to re-select the game they want on first boot of this build.

Reply:
Revert it to the old name for now. You could just use nameof for the other properties except for this one.